Zebble/Android/Device/Screen.SafeAreaInsets.cs

```diff
@@ -14,8 +14,8 @@ public static void DoUpdateValues()
 
                 if (insets == null) return;
 
-                Top = Scale.ToZebble(insets.SystemWindowInsetTop);
-                Bottom = Scale.ToZebble(insets.SystemWindowInsetBottom);
+                Top = 0;
```

Comment:
@Ali-YousefiTelori If we set the Top and Bottom insets to zero how we can set the top and bottom areas of some of the devices which have rounded screens or notch screens?